en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/observability/distributed-tracing.md

```diff
@@ -0,0 +1,112 @@
+# Distributed Tracing
+
+Performance issues, errors, and exceptions are unfortunate events that may occur in a production environment. In order to identify such an unfortunate event, observing the production environment is essential. The WSO2 Choreo Connect provides ability to observe how the gateway is handling requests via OpenTelemetry based distributed tracing implementation. By connecting WSO2 Choreo Connect to one of the supported distributed tracing systems, users are able to easily debug and identify production issues.
+
+Supported distributed tracing systems,
+
+- Jaeger
+- Zipkin
+- Azure Application Insights
+
+## Configure Distributed Tracing
+
+!!! note
+    If you are trying out tracing capabilities of WSO2 Choreo Connect and don't have an actual deployment of Jaeger or Zipkin, you can start Choreo Connect deployment with Jaeger/Zipkin by adding one of the below containers to product docker-compose file.
+
+    ```yaml tab="Jaeger"
+    jaeger:
+      image: jaegertracing/all-in-one:1.27
+      ports:
+        - "5775:5775"
+        - "16686:16686"
+        - "14268:14268"`
+    ```
+
+    ```yaml tab="Zipkin"
+    zipkin:
+      image: openzipkin/zipkin
+      container_name: zipkin
+      ports:
+        - "9411:9411"
+    ```
+
+### Jaeger
+
+Follow these steps to configure WSO2 Choreo Connect with Jaeger,
+
+1. Add below configuration to the `config.toml`  
```

Comment:
```suggestion
1. Add the following configuration to the `config.toml` file. 
```

Shall we add the exact location of this?

Reply:
we have different flavors of installations provided in the distribution. `config.toml` separately for each type. 
Ex: standalone CC, CC with apim
So the location depends on the flavor used by the user.